README.md

```diff
@@ -92,7 +92,7 @@ nuget restore OpenConsole.sln
 msbuild OpenConsole.sln
 ```
 
-We've provided a set of convenience scripts in the **/tools** directory to help automate the process of building and running tests.
+We've provided a set of convenience scripts as well as a README document in the **/tools** directory to help automate the process of building and running tests.

```

Comment:
```suggestion
We've provided a set of convenience scripts as well as [a README document](./tools/README.md) in the **/tools** directory to help automate the process of building and running tests.
```

Would allow people to simply click the URL to see that README

Reply:
I agree, it would make sense to link to the file if we're mentioning it.

Reply:
Thanks, made an update.